ksp/src/main/kotlin/fp/serrano/kopykat/KopyKatOptions.kt

```diff
@@ -3,15 +3,13 @@ package fp.serrano.kopykat
 internal data class KopyKatOptions(
   val copyMap: Boolean,
   val mutableCopy: Boolean,
-  val valueCopy: Boolean,
   val hierarchyCopy: Boolean
 ) {
   companion object {
     fun fromKspOptions(options: Map<String, String>) =
       KopyKatOptions(
         copyMap = options.parseBoolOrTrue("copyMap"),
         mutableCopy = options.parseBoolOrTrue("mutableCopy"),
-        valueCopy = options.parseBoolOrTrue("valueCopy"),
```

Comment:
Are the docs up-to-date with respect to the available options?

Reply:
No they weren't. Thank you :pray:   
670f29d